tests/func/test_gdrive.py

```diff
@@ -11,14 +11,18 @@
 
 class TestGDrive(TestDvc):
     def test(self):
-        main(["add", self.FOO, self.BAR, self.DATA, self.DATA_SUB])
         dirname = tempfile.mktemp("", "dvc_test_", "")
         url = "gdrive://root/" + dirname
-        main(["remote", "add", "gdrive", url])
-        main(["push", "-r", "gdrive"])
+
+        # check_call is used to produce correct test coverage
```

Comment:
I don't get this, could you elaborate?

Reply:
Adding files and configuring the remote is out of scope of this test, so it is better to run this commands via `check_call` so their execution wouldn't affect the test coverage information.

Reply:
I removed this comment, it is not useful here, actually.